lang/po/new/cs.po

```diff
@@ -1643,7 +1639,7 @@ msgstr "Komp. magnetu"
 # MSG_MESH c=12
 #: messages.c:147
 msgid "Mesh"
-msgstr ""
+msgstr "Mesh"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.